File: Backend/NodeJs-server/source/common/enums.ts
export enum Statuses {
    Active = 1,
    NotActive = 2,
}

export enum UserType {
    businessOwner = 1,
    socialActivist,
    nonProfitOrganization,
    prolobbyOwner,
    system
}

export enum Role {
    AccessAdministrator = 1,
    RegularUser,
    AreaManager,
    NetworkAdministrator,
    StoreManager,
    Cashier
}

// export const Role = {
//     Administrator: 1,
//     RegularUser: 2,
//     AccessAdministrator: 3,
//     NetworkAdministrator: 4,
//     StoreManager: 5,
//     Cashier: 6
// }
// type RoleType = typeof Role;

export enum AppError {
    General = "General",
    ConnectionError = "ConnectionError",
    QueryError = "QueryError",
    NoData = "NoData",
    NonNumericInput = "NonNumericInput",
    InputParameterNotSupplied = "InputParameterNotSupplied",
    DeletionConflict = "DeletionConflict"
}

File: Backend/dotNetBackend/MainService/Common/Enums.cs
namespace dotNetBackend.Common
{
    public enum Statuses 
    {
        Active = 1,
        NonActive = 2
    }

    public enum TransactionStates
    {
        Ordered = 1,
        Shipped = 2,
        Donated = 3
    }

    public enum UserTypes
    {
        BusinessOwner = 1,
        SocialActivist = 2,
        NonprofitOrganization = 3,
        ProlobbyOwner = 4,
        System = 5
    }
}
